src/Controller/OmniaController.php

```diff
@@ -141,282 +268,218 @@ public function videoList() {
    * @throws \Exception
    */
   public function video(Request $request) {
-
-    $response = new JsonResponse();
-    $content = $request->getContent();
-    $query = $this->mediaEntityStorage()->getQuery();
-
-    $token = $request->query->get('token', NULL);
-    if (NULL !== $token) {
-
-      $config = $this->config('nexx_integration.settings');
-      if ($token != $config->get('notification_access_key')) {
-        throw new AccessDeniedHttpException();
-      }
-    }
-    else {
+    if (!$this->isTokenValid($request)) {
+      $this->logger->error('Wrong or missing token in request.');
```

Comment:
Same as above: Do we really need the log entry? Could be exploited by anonymous users to generate mass log entries.